src/RawPOSIX/src/tests/fs_tests.rs

```diff
@@ -724,6 +724,20 @@ pub mod fs_tests {
         lindrustfinalize();
     }
 
+    #[test]
+    pub fn ut_lind_fs_mprotect_unmapped_addr() {
```

Comment:
good first test, lets add a few more, especially one where it splits an existing region